python/ray/log_monitor.py

```diff
@@ -121,7 +122,8 @@ def update_log_filenames(self):
         log_file_paths = glob.glob(f"{self.logs_dir}/worker*[.out|.err]")
         # segfaults and other serious errors are logged here
         raylet_err_paths = glob.glob(f"{self.logs_dir}/raylet*.err")
-        for file_path in log_file_paths + raylet_err_paths:
+        gcs_err_path = glob.glob(f"{self.logs_dir}/gcs_server.err")
```

Comment:
There's only one gcs server instance for the lifetime of clusters. This could be changed when we start supporting restart. Please let me know if you'd like me to address it here.

Reply:
Made it `gcs_server*.err`